subset/security/tlsfaux/server.py

```diff
@@ -28,22 +28,23 @@
 #OP_NO_TLSv1_2
 #OP_NO_TLSv1_3
 
-def do_something(connstream,data):
-	print "do_something:",data
+def parse_data(connstream,data):
+	print "parse_data:",data
```

Comment:
lint/style.

is this python3?  should be python3.

print("parse_data: ", data)

(spaces)